frontend/public/components/edit-yaml.jsx

```diff
@@ -685,6 +702,43 @@ export const EditYAML = connect(stateToProps)(
               <div className="co-p-has-sidebar">
                 <div className="co-p-has-sidebar__body">
                   <div className="yaml-editor" ref={(r) => (this.editor = r)}>
+                    <div className="yaml-editor__links">
+                      <div className="yaml-editor__link">
+                        <Popover
+                          aria-label="Shortcuts"
+                          bodyContent={
+                            <ul>
+                              <li>Use Ctrl + Space to active auto complete.</li>
+                              <li>Use Ctrl + Shift + O for document outlining.</li>
```

Comment:
I had trouble getting this work. I'm not sure if the keybinding is different on macOS.

Reply:
@bmignano - Would it be command + shift + o on Mac?

Reply:
Oh that's interesting. It looks like command + shift + o works on Mac for outlining, although ctrl + space works for auto completion.

Reply:
Updated text with "Command + Shift + O" for the meantime.

Reply:
@rebeccaalpert I'd check if `window.navigator.platform` starts with `Mac` to show the right shortcut